.github/workflows/codeforafrica-deploy-review-app.yml

```diff
@@ -0,0 +1,135 @@
+name: Code for Africa | Deploy | REVIEW APP
```

Comment:
Isn't this just part of `codeforafrica-deploy-dev.yml`?

Reply:
@kilemensi are you suggesting this file to be renamed to `codeforafrica-deploy-dev.yml`?

Reply:
Kind @koechkevin. Isn't this to be part of dev? i.e.

**Open/sync PR -> Deploy Review app cloned from Dev
Close/merge PR to main -> Deploy app Dev**

Of course, we can also clean these up in a separate PR if you think it's complex.

Reply:
Still think this should be part of dev. If you're still not sure, add a story to track it & we'll review sometime next week (set a reminder).